internal/skopeo_worker/main.go

```diff
@@ -138,6 +138,7 @@ func ProcessQueueMultiArch(commandFactory func(name string, arg ...string) exec_
 			size, err := downloadImageAndGetSize(imageName, architecture, tarballFilename, worker)
 			if err != nil {
 				worker.ErrorHandler.Handle(err)
+				worker.Rdb.LRem(worker.Ctx, worker.getProcessingQueueName(), 1, imageName).Result()
```

Comment:
maybe can this be defer'd? so it's automatically called regardless of the result?

Reply:
yes, i guess thats a better approach definitely :+1: